liquibase-core/src/main/java/liquibase/parser/core/java/JavaChangeLogParser.java

```diff
@@ -0,0 +1,61 @@
+package liquibase.parser.core.java;
+
+import liquibase.Scope;
+import liquibase.change.Change;
+import liquibase.changelog.ChangeLogParameters;
+import liquibase.changelog.DatabaseChangeLog;
+import liquibase.exception.ChangeLogParseException;
+import liquibase.parser.ChangeLogParser;
+import liquibase.resource.ResourceAccessor;
+
+import java.lang.reflect.Method;
+
+public class JavaChangeLogParser implements ChangeLogParser {
+
+    @Override
+    public DatabaseChangeLog parse(String physicalChangeLogLocation, ChangeLogParameters changeLogParameters, ResourceAccessor resourceAccessor) throws ChangeLogParseException {
+        try {
+            Class changeLogClass = Class.forName(physicalChangeLogLocation.replace("/", "."));
+            Object changeLogInstance = changeLogClass.newInstance();
+
+            DatabaseChangeLog changelog = new DatabaseChangeLog(physicalChangeLogLocation);
+
+            for (Method method : changeLogClass.getMethods()) {
+                ChangeSet changeSetAnnotation = method.getAnnotation(ChangeSet.class);
+                if (changeSetAnnotation != null && !changeSetAnnotation.ignore()) {
+                    String id = changeSetAnnotation.id();
+                    String author = changeSetAnnotation.author();
+                    boolean runAlways = changeSetAnnotation.runAlways();
+
+                    liquibase.changelog.ChangeSet changeSet = new liquibase.changelog.ChangeSet(id, author, runAlways, false, physicalChangeLogLocation, null, null, changelog);
```

Comment:
I don't like how we have both liquibase.changelog.ChangeSet and liquibase.parser.core.java.ChangeSet. Makes the code confusing. 

I'd tend to think we should have a different name for the annotation since that is a bit less "core" than the ChangeSet object, but we also want the annotated java class to be as readable as possible? Any ideas for a better name for the annotation? Maybe something that would work for changesSet, include, and other methods we may want? (See the example changelog below)